src/triage/api/schema.py

```diff
@@ -0,0 +1,232 @@
+import logging
+import pathlib
+import hashlib
+import graphene
+from graphene_file_upload.scalars import Upload
+
+import graphql_jwt
+from graphql_jwt.decorators import login_required
+
+from django.core.exceptions import ValidationError
+from packageurl import PackageURL
+from triage.models import ProjectVersion
+from triage.util.finding_importers.archive_importer import ArchiveImporter
+
+logger = logging.getLogger(__name__)
+MAX_FILE_SIZE_BYTES = 200 * 1024 * 1024  # Allowed file size: 200 MB in bytes
```

Comment:
Maybe split off 200 into another variable to make it easier to edit should it be too small. Not super important, but could be a nice touch